core/templates/dev/head/pages/landing/fractions/landing_page_teacher.html

```diff
@@ -7,243 +7,282 @@
+              <br><br>We’ll make sure they’re not confused along the way with helpful hints at every step.
+              </h2>
+            </div>
+          </div>
+        </div>
       </div>
     </div>
 
     <div class="oppia-fractions-landing-section text-center">
-      <div style="position: relative; margin-left: auto; margin-right: auto; max-width: 2000px; height: 560px">
-        <h1 class="oppia-fractions-landing-h1" style="color: #242424; padding-top: 114px; text-align: right; padding-right: 83px">Easy to follow lessons</h1>
-        <h2 class="oppia-fractions-landing-h2" style="color: #242424; top: 188px; text-align: right; right: 83px">With Oppia, we make it easy to go through the lessons and learn something new! Follow Matthew, as he works hard to become a great baker!
-          <br><br>Students get to work through stories and apply their knowledge of real-world problems.
-        </h2>
-        <img ng-src="<[getStaticImageUrl('/fractions_landing/student/lessons_lessons.svg')]>" class="oppia-fractions-landing-image" style="left: 80px; top: 40px; width: 352px; z-index: 20" alt="">
-        <img ng-src="<[getStaticImageUrl('/fractions_landing/student/storytelling_matthew.svg')]>" class="oppia-fractions-landing-image" style="left: 260px; top: 310px; width: 212px; z-index: 30" alt="">
-        <img ng-src="<[getStaticImageUrl('/fractions_landing/student/storytelling_pizza.svg')]>" class="oppia-fractions-landing-image" style="left: 340px; top: 241px; width: 255px; z-index: 10" alt="">
-        <img ng-src="<[getStaticImageUrl('/fractions_landing/student/storytelling_dessert.svg')]>" class="oppia-fractions-landing-image" style="left: 415px; top: 330px; width: 255px; z-index: 20" alt="">
+      <div class="oppia-fractions-landing-section-inner">
+        <h1 class="oppia-fractions-landing-h1" style="color: #242424; text-align: right; padding-right: 10%">Easy to follow lessons</h1>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-sm-6 oppia-fractions-landing-image-div">
+              <img ng-src="<[getStaticImageUrl('/fractions_landing/teachers/storytelling_teacher_combined.png')]>" class="oppia-fractions-landing-image oppia-fractions-landing-image-desktop" alt="">
+              <img ng-src="<[getStaticImageUrl('/fractions_landing/student/lessons_lessons.svg')]>" class="oppia-fractions-landing-image oppia-fractions-landing-image-mobile" alt="">
+            </div>
+            <div class="col-sm-6">
+              <h2 class="oppia-fractions-landing-h2" style="color: #242424; text-align: right; right: 10%">With Oppia, we make it easy to go through the lessons and learn something new!
+                <br><br>You will work to learn new concepts and help Matt become a great baker along the way. We make it fun and easy to get started.
```

Comment:
Just realized something is off about the text on this page. Suggest:

You will work ... --> Students will work ...
Matt --> Matthew

(edit: please check the mocks again, this text is not correct)

Reply:
I had missed this when I finally pasted the student page to the teacher page. Fixed it now. Sorry about that